recipes/dd-agent.rb

```diff
@@ -103,3 +103,6 @@
 
 # Install integration packages
 include_recipe 'datadog::integrations' unless is_windows
+
+# Install custom checks
+include_recipe 'datadog::custom_checks' unless is_windows # FIXME: implement this on windows?
```

Comment:
I think Windows support could be as easy adding a `if` on `cookbook_file` args
```ruby
owner 'dd-agent'
mode '644'
```
and use the Windows equivalent in this case. Anyway, we can target the next release.